### DISCUSSION TITLE
Has Anyone Attempted using qemu-system-aarch64 as the hypervisor on an x86 host?

### DISCUSSION THREAD

**Discussion by DJ-Architect** [kata-containers/kata-containers#6121] [Q&A]
I've been trying to attempt this to no avail so I'm partially inclined to believe it isn't supported. I've been trying to utilize the qemu-system-aarch64 hypervisor which resulted in errors during sandbox creation where the TCG property could not be located. kvm acceleration won't work since this is cross architecture emulation. Has anyone attempted this before?

**Comment by fidencio** [comment]
Nops, and Kata Containers actually relies on KVM.  So I'd say that's not a supported case.
